README.md

```diff
@@ -1,26 +1,98 @@
 # Loop Exercises
 
 1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
-2. Write a `while` loop and a `for` loop that takes the variable "num", and iterates over all numbers from 0 to "num". 
+
+let num = 10
+while (num >= 1) {
+  console.log('the number is: ' + num)
+  num -= 1
+}
+
+
+2. Write a `while` loop and a `for` loop that takes the variable "num", and iterates over all numbers from 0 to "num".
 For each iteration, it will check if the current number is even or odd, and log that to the screen (e.g. "2 is even")
-3. Write a `while` loop and a `for` loop that takes the variable "num" and iterates over all numbers from 0 to "num". 
+
+num = 10
+for (let i = 0; i <= num; i++ )
+  if (i % 2 === 0 ) {
+  console.log( i + " is even" )
+} else {
+console.log(i + 'is odd') }
+
+
+3. Write a `while` loop and a `for` loop that takes the variable "num" and iterates over all numbers from 0 to "num".
 For each iteration of the loop, it will multiply the number by 9 and log the result (e.g. "2 * 9 = 18").
 
-_Bonus_ think of another way to solve it. 
+_Bonus_ think of another way to solve it.
   <details>
     <summary>
       Hint
     </summary>
     Find the final number and increment the loop by 9.
   </details>
-  
+
+  let num = 0
+  while (num <= 10) {
+    console.log(num * 9)
+    num += 1
+  }
+
+  for (let i = 0; i <= 10; i++) {
+    console.log(i*9)
+  }
+
 4. Write a loop that uses console.log to log all the numbers from 1 to 100, with two exceptions. For numbers divisible by 3, log "Fizz" instead of the number, and for numbers divisible by 5 (and not 3), log "Buzz" instead.
 
+for (let i = 1; i <= 100; i++){if
+(i % 3 === 0) {
+  console.log("fizz")
+} if (i % 5 === 0 && i % 3 > 0) {
```

Comment:
The extra %3 logic isn't actually necessary here - you have captured everything with `i % 5 === 0`.